api/src/main/java/org/pmiops/workbench/api/UserMetricsController.java

```diff
@@ -167,7 +167,7 @@ public void setDistinctWorkspaceLimit(int limit) {
 
   /** Gets the list of all resources recently access by user in order of access date time */
   @Override
-  public ResponseEntity<RecentResourceResponse> getUserRecentResources() {
+  public ResponseEntity<WorkspaceResourceResponse> getUserRecentResources() {
```

Comment:
If it's not too late, could you please split out the rename refactorings into a follow-on PR? It's a bit difficult to follow everything that's happening.

Reply:
Sure. I can try to do that. We will have a bit of a weird in-between state if that's okay?